src/js/listeners/post/createListing.js

```diff
@@ -1,41 +1,27 @@
+// Co-author: Oskar Jenssen
 import { create } from "../../api/posts/create.js";
 
 /**
  * A listener for when users are creating a new listing
  */
 export async function createListing() {
   const form = document.querySelector("#createNewListing");
-  const createTitle = document.querySelector("#createTitle");
-  const createLocation = document.querySelector("#createLocation");
-  const createDeadline = document.querySelector("#createDeadline");
-  const createPicture = document.querySelector("#pictureUrl");
-  const createDescription = document.querySelector("#createDescription");
 
   if (form) {
     form.addEventListener("submit", async (event) => {
       event.preventDefault();
 
-      const title = createTitle.value;
-      const location = createLocation.value;
-      const deadline = createDeadline.value;
-      const picture = createPicture.value;
-      const desc = createDescription.value;
-      const currentTime = new Date().toISOString();
+    const form = event.target;
```

Comment:
I'm not able to test this yet due to pending fixes - but this doesn't sound correct because there is already a const form declared in line 8